### ISSUE TITLE
[UI] Displaying text

### ISSUE THREAD

**Issue by guglovich** [benjamimgois/goverlay#5]
Waited for the upgrade to 0.9, saw the item about adapting the theme to the system theme. However, the fonts as in the last version I have are not congruent, grey on top of the white tab header. 

![изображение](https://user-images.githubusercontent.com/54148165/180246204-2714e3a9-acfa-48a3-b2ef-78b5998875dd.png)

**Comment by arcDaniel84** [comment]
I'm to have some "gliches" with the Breeze theme.
![image](https://user-images.githubusercontent.com/67470462/182185523-9d772d51-24ec-4b1c-802c-af2509809d5b.png)
![image](https://user-images.githubusercontent.com/67470462/182185665-89b59d98-031f-4216-88b5-1fa186c9f03e.png)

**Comment by benjamimgois** [comment]
> Waited for the upgrade to 0.9, saw the item about adapting the theme to the system theme. However, the fonts as in the last version I have are not congruent, grey on top of the white tab header.
> 

That tab colors gave me a hard time in the last release. I'll try to improve the situation in next point release.

**Comment by benjamimgois** [comment]
> I'm to have some "gliches" with the Breeze theme. ![image](https://user-images.githubusercontent.com/67470462/182185523-9d772d51-24ec-4b1c-802c-af2509809d5b.png) ![image](https://user-images.githubusercontent.com/67470462/182185665-89b59d98-031f-4216-88b5-1fa186c9f03e.png)

Are you using X11 with desktop scalling ? Using desktop scalling is breaking qt5pas interface on X11, but works on wayland.

**Comment by arcDaniel84** [comment]
Yes I use X11 with 125% Scaling. Sadly I find Wayland on KDE not ready with the Scaling. A lot of App are really blurry. So here I prefer a little glitchy interface with GOverlay, witch I open rarely, that a blurry Firefox.

but thx, for replying and thanks, for you wonderful work

**Comment by benjamimgois** [comment]
> Yes I use X11 with 125% Scaling. Sadly I find Wayland on KDE not ready with the Scaling. A lot of App are really blurry. So here I prefer a little glitchy interface with GOverlay, witch I open rarely, that a blurry Firefox.


I already openned a issue in lazarus about this scaling problem on x11. I guess we'll have to wait for a fix, but since it works fine on wayland i don't think it's a hight priority bug for them. (I also have this issue on my main game machine, since nvidia is still buggy on wayland).

> but thx, for replying and thanks, for you wonderful work

Thanks for the kind words

**Comment by arcDaniel84** [comment]
With the last version the problem is completely solved, thanks!!
